lib/rubocop/cop/chef/effortless/depends_chef_vault.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+#
+# Copyright:: 2019, Chef Software Inc.
```

Comment:
```suggestion
# Copyright:: 2020, Chef Software Inc.
```

Reply:
Might as well toss an author block up there as well to give yourself some street cred